tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "@tsconfig/recommended/tsconfig.json",
   "compilerOptions": {
+    "target": "ES2020",
+    "moduleResolution": "NodeNext",
```

Comment:
Why add those?

Reply:
now, the target is ES2015. The higher target, the code is better performance. because the less syntax be transformed. For example the async function, es2020 native support async  function, but in es2015, the async function will be transform ed to generators. 

As for moduleResolution: nodeNext, this is a more mordern resolution enable us to use the pavkage.json exports field.